src/Symfony/Bundle/FrameworkBundle/Resources/config/web.php

```diff
@@ -116,6 +116,7 @@
                 param('kernel.enabled_locales'),
             ])
             ->tag('kernel.event_subscriber')
+            ->tag('kernel.reset', ['method' => 'reset', 'on_invalid' => 'ignore'])
```

Comment:
this should be set on the `request_stack` service, not on the `locale_listener` service.

Reply:
oops, fixed